### ISSUE TITLE
Something related to API is broken in release branch

### ISSUE THREAD

**Issue by abitmore** [bitshares/bitshares-core#979]
**Bug Description**
bitCNY doesn't show in my old light wallet (160420), actually all bitCNY related things e.g. account history/asset info/price/feed/market don't show. Blank.

bitUSD shows only once in account history.

If switched to a node running 2.0.180328, all become fine.

**Comment by abitmore** [comment]
I think it's related to jsonification change of `call_order_update` operation.

**Comment by clockworkgr** [comment]
160420? seriously? :D

**Comment by abitmore** [comment]
Or perhaps due to new "asset_id" field added in `asset_bitasset_data_object`.

**Comment by abitmore** [comment]
At least `asset_id` field of bitCNY is `1.3.0` which is wrong. We need to fix it. Creating another issue.

**Comment by abitmore** [comment]
Looks like #981 fixed this issue.
